prefetch.bs

```diff
@@ -856,6 +867,7 @@ The <dfn>list of sufficiently strict speculative navigation referrer policies</d
         1. If |prefetchRecord|'s [=prefetch record/had conflicting credentials=] is true, then set |navigationParams| to null.
 
             <div class="note">This means that if any cross-partition origin along the redirect chain had credentials (and did not override this behavior using [:Supports-Loading-Mode:]), the prefetch is discarded. This reduces the chance of the user observing a logged-out page when they are logged in.</div>
+        1. [=Trigger a prefetch status updated event=] given |document|'s [=node navigable=], |prefetchRecord|, and "{^speculation.PreloadingStatus/pending^}" status.
```

Comment:
This seems a bit too late to me, as it waits until after the fetch.

Probably we want to do this before creating the navigation params by fetching?

Reply:
I agree. I've moved it to after we start the time to expire timer, which is about where it is in our implementation as well.